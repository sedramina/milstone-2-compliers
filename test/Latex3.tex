\documentclass{article}
\usepackage[utf8]{inputenc}



\begin{document}
\title{Third Template}
\date{March 2016}
\maketitle

\section{Item List and Equations}
In the item list the body of the item list is either a plain text or a text displayed in math mode. \\
\\
\begin{itemize}
\item item 1
\item $1+3$
\item item $2$
\item item 3
\item item 4
\end{itemize}


%This line is commented and it should not be shown in your output

\section{Missing Item List}
In latex an item list can not be empty. If so a compilation error will occur stating the problem. Similar to template $2$ the line number should appear in the error message as well.

\begin{itemize}

\end{itemize}



\end{document}