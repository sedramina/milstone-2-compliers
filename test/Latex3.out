Third Template
March 20161  
Item List and Equations
In the item list the body of the item list is either a plain text or a text displayed in math mode.


item 1
1 + 3
item 2
item 3
item 4
Missing Item List
In latex an item list can not be empty.  If so a compilation error will occur stating the problem. Similar to template 2 the line number should appear inthe error message as well.
Parse Error
